service/billing/model.go

```diff
@@ -179,7 +179,15 @@ type DeleteBudgetRequest struct {
 	BudgetId string `json:"-" url:"-"`
 }
 
-// This describes an enum
+// The status string for log delivery. Possible values are: * `CREATED`: There
```

Comment:
@pietern This is a result of the changes in https://github.com/databricks/universe/pull/380074. We should do some post-processing in the SDK generators to replace this comment. 

Previously, we replaced the description with `This describes an enum` which led to an information loss. We need to retain the description for the API reference.

Reply:
Any idea why some of the other enums in this diff do contain newlines and this one doesn't?

Reply:
I was looking into this as well. Its a weird behavior. Let me check what's happening here